### ISSUE TITLE
[Feature request] Can you please add all the keys from a full-sized PC keyboard?

### ISSUE THREAD

**Issue by alpharesearch** [futo-org/android-keyboard#264]
It doesn't even have to be many extra keys, as long as I could select everything with long press, for example for F1 on hold 1 and F2 would be on 2 etc. You could build up the arrow keys on WASD for example. Also, there is still a lot of space left under the ?123 key. But some modifier buttons like control or alt or tab would need to have its own key. My use case is SSH and VNC.

**Comment by kj4ezj** [comment]
I am not against this, but apps like [Termux](https://f-droid.org/packages/com.termux/) (FOSS)...

![Termux](https://github.com/user-attachments/assets/85b39875-6c84-4222-9d51-59b6721a8ec5)

...and [JuiceSSH](https://play.google.com/store/apps/details?id=com.sonelli.juicessh) (non-free)...

![JuiceSSH](https://github.com/user-attachments/assets/c8e01b23-775c-46cd-83c7-b78229e3e3bd)

...do not expect keyboards to provide these keys and provide some of them for you.

**Comment by alpharesearch** [comment]
Agreed, but I have a Proxmox installation, and sometimes I use the built-in web VNC. Hacker's Keyboard is my go-to in this case; it's just less clunky than the built-in solutions. I do have the pro version of JucySSH, and as soon as I can pay for Futo via Google Play, I will. The voice on the Futo keyboard is superior to anything, maybe because I'm not a native English speaker. I like the swipe motion on the Samsung keyboard spacebar to switch between languages, and I do this quite a bit. This is much faster than a menu and maybe another thing on my wishlist.

**Comment by KamWithK** [comment]
How would you go about adding an extra row for arrows and modifiers?
I'm interested in contributing to add this feature
